cmd/fuzzbinder/main.go

```diff
@@ -13,13 +13,60 @@
 // limitations under the License.
 
 // Package main contains a command-line tool for generating fuzzing claims
-// for a revision of the source code.
+// for a revision of a source code.
 package main
 
 import (
+	"encoding/json"
+	"flag"
 	"log"
+	"os"
+	"path/filepath"
+
+	"github.com/project-oak/transparent-release/internal/fuzzbinder"
 )
 
 func main() {
-	log.Println("Not yet implemented!")
+	fuzzParameters := &fuzzbinder.FuzzParameters{}
+	flag.StringVar(&fuzzParameters.ProjectName, "project_name", "",
+		"Required - Project name as defined in OSS-Fuzz projects.")
+	flag.StringVar(&fuzzParameters.ProjectGitRepo, "git_repo", "",
+		"Required - GitHub repository of the project.")
+	// TODO(#175): Remove fuzzEngine and sanitizer from FuzzBinder inputs.
+	flag.StringVar(&fuzzParameters.FuzzEngine, "fuzzengine", "libFuzzer",
+		"Required - Fuzzing engine used for the project. Examples: libFuzzer, afl, honggfuzz, centipede.")
+	flag.StringVar(&fuzzParameters.Sanitizer, "sanitizer", "asan",
+		"Required - Fuzzing sanitizer used for the project. Examples: asan, ubsan, msan.")
+	flag.StringVar(&fuzzParameters.Date, "date", "",
+		"Required - Fuzzing date. The expected date format is YYYYMMDD. \n"+
+			"This date must not be more than 15 days prior to the date of execution of this cmd. \n"+
```

Comment:
> Why? What happens if the date is too far in the past or even future?

In the future: 
You will get an error `could not generate the fuzzing claim: could not get the revision digest to generate the fuzzing claim: could not get oak/srcmap/20221220.json blob to extract revision hash: could not create a new reader for blob "oak/srcmap/20221220.json": storage: object doesn't exist`

In the past: 
You will get a FuzzClaim generated but with no fuzzing efforts and the "detectedCrashes" will be `false` (but it shouldn't be trusted because the log files are  deleted). You will also have the coverage data and it is reliable (because the coverage reports are kept for years).

Reply:
> I suggest removing this from the help,
Done !

Reply:
I added an issue for this and added it as TODO in the code.